source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -37,13 +38,17 @@ Qonto wrapped up the code in a small addon, [`ember-autofocus-modifier`](https:/
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## 3 New RFCs To Add New Template Helpers 3️⃣
```

Comment:
The goodbitsing newsletter tool currently expects each title to also have an url present. For example, we could link the RFC repos with all of Miguel's relevant template helper PRs that are mentioned in this section. What do you think?

```suggestion
## [3 New RFCs To Add New Template Helpers 3️⃣](https://github.com/emberjs/rfcs/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+author%3Acibernox+templates)
```

Reply:
Oh this is awesome didnt think about this!